### PR TITLE
ForgotPassword Component Story

### PR DESCRIPTION
**Pull request by Faria-Ejaz**

### CODE REVIEW

File: client/src/stories/ForgotPassword.stories.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import ForgotPassword from '../containers/ForgotPassword/ForgotPassword';
+import { Provider } from 'react-redux';
+import configureStore from '../store/configureStore';
+
+const store = configureStore();
+
+storiesOf('Forgot Password', module).add('Enter Email', () => {
```
Comment (mashhoodr) [inline client/src/stories/ForgotPassword.stories.js]: @Faria-Ejaz a good start, the idea behind story book is to show the different state of each component. So lets pick a component which has several states and document that.

File: client/src/stories/EventCard.stories.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import EventCard from '../components/EventList/EventCard/EventCard';
+import { BrowserRouter, Route } from 'react-router-dom';
+
+const event = {
```
Comment (taleymirza) [inline client/src/stories/EventCard.stories.js]: make a file for all data used in tests

File: client/src/stories/SocialShare.stories.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      </FacebookShareButton>
+      <TwitterShareButton className="mx-1" url={window.location.href}>
+        <TwitterIcon size={32} round={true} />
+      </TwitterShareButton>
+    </div>
+  );
+  return story;
+});
+
+storiesOf('SocialShare', module).add('right vertical', () => {
```
Comment (taleymirza) [inline client/src/stories/SocialShare.stories.js]: what is difference between above and this story?

Comment (taleymirza) [inline client/src/stories/SocialShare.stories.js]: what is difference between above and this story?

File: client/src/stories/CommentList.stories.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+//     username: 'faria',
+//     comment: "helllo",
+//     avatar: 'https://pay.google.com/about/static/images/social/knowledge_graph_logo.png',
+//     commentDatetime: '22-12-2018'
+// };
+
+// storiesOf('CommentList', module).add('default', () => {
+//   const story = <CommentList {..props}/>;
+//   return story;
+// });
```
Comment (taleymirza) [inline client/src/stories/CommentList.stories.js]: why you comment this?

Comment (taleymirza) [inline client/src/stories/CommentList.stories.js]: why you comment this?

Comment (taleymirza) [inline client/src/stories/CommentList.stories.js]: why you comment this?

**Comment by mashhoodr** [comment]
Closing this PR since it was just for practice.

**Pull request closed without merging**